include/SongLoaderBen.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include <string>
+#include <stdlib.h>
+#include <dirent.h>
+#include <errno.h>
+#include <iostream>
+#include <stdio.h>
+
+//headers for randomizing
+#include <algorithm>
+#include <vector>
+#include <random>
+
+class SongLoaderBen
+{
+private:
+    std::vector<std::string> files =std::vector<std::string>();
+    int currentSongIndex;
+    bool checkLibrary;
+    std::vector<std::string> songList;
+    std::string CurrentSong;
+    bool readSongNames(std::string dir, std::vector<std::string> &files);
+
+public:
+    SongLoaderBen();
+    bool nextSong();
+    bool previousSong();
```

Comment:
nextSong() and previousSong() need to return std::string's.
They have similar functions compared to gotoNextSong, however, they do not actually move the `currentSongIndex` anywhere.